DESCRIPTION

```diff
@@ -24,24 +24,24 @@ Imports:
   Rcpp,
   magrittr,
   dplyr,
-  reshape2,
   data.table,
   ggplot2,
   plotly,
   colorspace,
   colorRamps,
   RColorBrewer,
   shiny,
-  shinyjs,
-  shinydashboard,
-  markdown,
-  DBI,
-  dbplyr,
   withr
 LinkingTo: Rcpp
 SystemRequirements: C++11
 RoxygenNote: 6.1.1
 Suggests: 
     testthat
+Enhances:
+    shinydashboard,
+    markdown,
```

Comment:
Oh, I just noticed that I misunderstood what enhances does. I will remove this (I removed it from imports because of the notes we got from CRAN

Reply:
Just to elaborate on this: the IOHanalyzer package does not export any functions which use imports form these packages. This generates a note in CRAN-tests (only on some builds), so it might be best to remove them for now. Maybe we can move them to suggests? 
Or maybe just move them back to imports again and assume CRAN sees that they are needed for the GUI?